api/api.go

```diff
@@ -26,20 +26,22 @@ type Api struct {
 	MemberID   string
 	Identifier string
 	MWHandler  *rye.MWHandler
+	DebugUI    bool
 }
 
 type JSONStatus struct {
 	Status  string
 	Message string
 }
 
-func New(cfg *config.Config, mwHandler *rye.MWHandler, version string) *Api {
+func New(cfg *config.Config, mwHandler *rye.MWHandler, version string, debugUI bool) *Api {
```

Comment:
This "flag" that is passed in through main allows you to choose whether you're running against the compiled `statik.go` file or you are running through serving the file system directly (the static JS app files).